### PR TITLE
Move to E104 EG51 WBPS15 EP51

### PR DESCRIPTION
**Pull request by a-solovyev12**

### CODE REVIEW

File: sh/gtf/gxa_references.conf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 aegilops_tauschii ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/aegilops_tauschii/Aegilops_tauschii.Aet_v4.0.RELNO.gtf.gz
 amborella_trichopoda ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/amborella_trichopoda/Amborella_trichopoda.AMTR1.0.RELNO.gtf.gz
 anas_platyrhynchos ftp://ftp.ensembl.org/pub/release-RELNO/gtf/anas_platyrhynchos_platyrhynchos/Anas_platyrhynchos_platyrhynchos.CAU_duck1.0.RELNO.gtf.gz
-anolis_carolinensis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/anolis_carolinensis/Anolis_carolinensis.AnoCar2.0.RELNO.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf:4]: RELNO should not be changed for an explicit value here.

File: sh/gtf/gxa_references.conf [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+anolis_carolinensis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/anolis_carolinensis/Anolis_carolinensis.AnoCar2.0v2.104.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf]:
```suggestion
anolis_carolinensis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/anolis_carolinensis/Anolis_carolinensis.AnoCar2.0v2.RELNO.gtf.gz
```

File: sh/gtf/gxa_references.conf [hunk 3]
```diff
@@ -12,8 +12,14 @@ brachypodium_distachyon ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gt
 brassica_oleracea ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/brassica_oleracea/Brassica_oleracea.BOL.RELNO.gtf.gz
 brassica_rapa ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/brassica_rapa/Brassica_rapa.Brapa_1.0.RELNO.gtf.gz
 brassica_napus ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/brassica_napus/Brassica_napus.AST_PRJEB5043_v1.RELNO.gtf.gz
+chenopodium_quinoa ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/chenopodium_quinoa/Chenopodium_quinoa.ASM168347v1.51.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf]:
```suggestion
chenopodium_quinoa ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/chenopodium_quinoa/Chenopodium_quinoa.ASM168347v1.RELNO.gtf.gz
```

File: sh/gtf/gxa_references.conf [hunk 4]
```diff
@@ -24,7 +30,7 @@ cyanidioschyzon_merolae ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gt
 danio_rerio ftp://ftp.ensembl.org/pub/release-RELNO/gtf/danio_rerio/Danio_rerio.GRCz11.RELNO.gtf.gz
 dasypus_novemcinctus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/dasypus_novemcinctus/Dasypus_novemcinctus.Dasnov3.0.RELNO.gtf.gz
 dictyostelium_discoideum ftp://ftp.ensemblgenomes.org/pub/protists/release-RELNO/gtf/dictyostelium_discoideum/Dictyostelium_discoideum.dicty_2.7.RELNO.gtf.gz
-drosophila_melanogaster ftp://ftp.ensemblgenomes.org/pub/release-RELNO/metazoa/gtf/drosophila_melanogaster/Drosophila_melanogaster.BDGP6.28.RELNO.gtf.gz
+drosophila_melanogaster ftp://ftp.ensemblgenomes.org/pub/release-RELNO/metazoa/gtf/drosophila_melanogaster/Drosophila_melanogaster.BDGP6.32.51.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf]:
```suggestion
drosophila_melanogaster ftp://ftp.ensemblgenomes.org/pub/release-RELNO/metazoa/gtf/drosophila_melanogaster/Drosophila_melanogaster.BDGP6.32.RELNO.gtf.gz
```

File: sh/gtf/gxa_references.conf [hunk 5]
```diff
@@ -42,10 +48,10 @@ leersia_perrieri ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/leers
 macaca_mulatta ftp://ftp.ensembl.org/pub/release-RELNO/gtf/macaca_mulatta/Macaca_mulatta.Mmul_10.RELNO.gtf.gz
 medicago_truncatula ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/medicago_truncatula/Medicago_truncatula.MedtrA17_4.0.RELNO.gtf.gz
 monodelphis_domestica ftp://ftp.ensembl.org/pub/release-RELNO/gtf/monodelphis_domestica/Monodelphis_domestica.ASM229v1.RELNO.gtf.gz
-musa_acuminata ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/musa_acuminata/Musa_acuminata.ASM31385v1.RELNO.gtf.gz
-mus_musculus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/mus_musculus/Mus_musculus.GRCm38.RELNO.gtf.gz
+musa_acuminata ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/musa_acuminata/Musa_acuminata.musa_acuminata_v2.51.gtf.gz
+mus_musculus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/mus_musculus/Mus_musculus.GRCm39.104.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf]:
```suggestion
musa_acuminata ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/musa_acuminata/Musa_acuminata.musa_acuminata_v2.RELNO.gtf.gz
mus_musculus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/mus_musculus/Mus_musculus.GRCm39.RELNO.gtf.gz
```

File: sh/gtf/gxa_references.conf [hunk 6]
```diff
@@ -42,10 +48,10 @@ leersia_perrieri ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/leers
...
 oryctolagus_cuniculus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/oryctolagus_cuniculus/Oryctolagus_cuniculus.OryCun2.0.RELNO.gtf.gz
-ornithorhynchus_anatinus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/ornithorhynchus_anatinus/Ornithorhynchus_anatinus.OANA5.RELNO.gtf.gz
+ornithorhynchus_anatinus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/ornithorhynchus_anatinus/Ornithorhynchus_anatinus.mOrnAna1.p.v1.104.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf]:
```suggestion
ornithorhynchus_anatinus ftp://ftp.ensembl.org/pub/release-RELNO/gtf/ornithorhynchus_anatinus/Ornithorhynchus_anatinus.mOrnAna1.p.v1.RELNO.gtf.gz
```

File: sh/gtf/gxa_references.conf [hunk 7]
```diff
@@ -77,11 +83,11 @@ solanum_tuberosum ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/sola
 sorghum_bicolor ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/sorghum_bicolor/Sorghum_bicolor.Sorghum_bicolor_NCBIv3.RELNO.gtf.gz
 sus_scrofa ftp://ftp.ensembl.org/pub/release-RELNO/gtf/sus_scrofa//Sus_scrofa.Sscrofa11.1.RELNO.gtf.gz
 tetraodon_nigroviridis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/tetraodon_nigroviridis/Tetraodon_nigroviridis.TETRAODON8.RELNO.gtf.gz
-theobroma_cacao ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/theobroma_cacao/Theobroma_cacao.Criollo_cocoa_genome_V2.RELNO.gtf.gz
+theobroma_cacao ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/theobroma_cacao/Theobroma_cacao.Theobroma_cacao_20110822.51.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf]:
```suggestion
theobroma_cacao ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/theobroma_cacao/Theobroma_cacao.Theobroma_cacao_20110822.RELNO.gtf.gz
```

File: sh/gtf/gxa_references.conf [hunk 8]
```diff
@@ -77,11 +83,11 @@ solanum_tuberosum ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/sola
 sorghum_bicolor ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/sorghum_bicolor/Sorghum_bicolor.Sorghum_bicolor_NCBIv3.RELNO.gtf.gz
 sus_scrofa ftp://ftp.ensembl.org/pub/release-RELNO/gtf/sus_scrofa//Sus_scrofa.Sscrofa11.1.RELNO.gtf.gz
 tetraodon_nigroviridis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/tetraodon_nigroviridis/Tetraodon_nigroviridis.TETRAODON8.RELNO.gtf.gz
-theobroma_cacao ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/theobroma_cacao/Theobroma_cacao.Criollo_cocoa_genome_V2.RELNO.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf:80]: @pinin4fjords could you check if ISL is using Theobroma cacao criollo or Theobroma cacao with no variety set? Thanks. This changed happened a while back when E! Plants moved to criollo as default, but they have again reinstated the none criollo one.

File: sh/gtf/gxa_references.conf [hunk 9]
```diff
@@ -77,11 +83,11 @@ solanum_tuberosum ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/sola
...
+theobroma_cacao ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/theobroma_cacao/Theobroma_cacao.Theobroma_cacao_20110822.51.gtf.gz
 trifolium_pratense ftp://ftp.ensemblgenomes.org/pub/plants/release-RELNO/gtf/trifolium_pratense/Trifolium_pratense.Trpr.RELNO.gtf.gz
 triticum_aestivum ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/triticum_aestivum/Triticum_aestivum.IWGSC.RELNO.gtf.gz
 triticum_urartu ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/triticum_urartu/Triticum_urartu.ASM34745v1.RELNO.gtf.gz
 vitis_vinifera ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/vitis_vinifera/Vitis_vinifera.12X.RELNO.gtf.gz
 xenopus_tropicalis ftp://ftp.ensembl.org/pub/release-RELNO/gtf/xenopus_tropicalis/Xenopus_tropicalis.Xenopus_tropicalis_v9.1.RELNO.gtf.gz
 yarrowia_lipolytica ftp://ftp.ensemblgenomes.org/pub/fungi/release-RELNO/gtf/yarrowia_lipolytica/Yarrowia_lipolytica.GCA_000002525.1.RELNO.gtf.gz
-zea_mays ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/zea_mays/Zea_mays.B73_RefGen_v4.RELNO.gtf.gz
+zea_mays ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/zea_mays/Zea_mays.Zm-B73-REFERENCE-NAM-5.0.51.gtf.gz
```
Comment (pcm32) [inline sh/gtf/gxa_references.conf]:
```suggestion
zea_mays ftp://ftp.ensemblgenomes.org/pub/release-RELNO/plants/gtf/zea_mays/Zea_mays.Zm-B73-REFERENCE-NAM-5.0.RELNO.gtf.gz
```

File: sh/gtf/download_gtfs.sh [hunk 10]
```diff
@@ -16,7 +16,7 @@ WBPS_REL=$3
 # The template to slot release numbers into
 gxaRefsTemplate=$(dirname ${BASH_SOURCE[0]})/gxa_references.conf
 # The config file that will be used for the actual download of the gtf files
-gxaRefs=~/tmp/gxa_references.conf.`eval date +%Y-%m-%d`.$$
+gxaRefs=/tmp/gxa_references.conf.`eval date +%Y-%m-%d`.$$
```
Comment (pinin4fjords) [inline sh/gtf/download_gtfs.sh]: Want to remove the backticks?

File: sh/gtf/gxa_references.conf (hunk 8 above)
Comment (pinin4fjords) [reply sh/gtf/gxa_references.conf:80]: Current 'active' configuration in ISL is:

```
reference=Theobroma_cacao.Criollo_cocoa_genome_V2.dna.toplevel.fa.gz
cdna_file=Theobroma_cacao.Criollo_cocoa_genome_V2.cdna.all.44.fa.gz
gtf_file=Theobroma_cacao.Criollo_cocoa_genome_V2.44.gtf.gz
```

... but I'll check the config file to see what's being downloaded.

File: sh/gtf/download_gtfs.sh (hunk 10 above)
Comment (pcm32) [reply sh/gtf/download_gtfs.sh]:
```suggestion
gxaRefs=/tmp/gxa_references.conf.$( eval date +%Y-%m-%d ).$$
```

**Review by pcm32 — APPROVED** [review]
I have checked though all files, there are unfortunately organism's family and gene symbol losses, but that is how it is (this really doesn't depend on us).

**Comment by pcm32** [comment]
For future record, the following species lost ensembl family on this go:
- anas_platyrhynchos
- bos_taurus
- callithrix_jacchus
- canis_familiaris
- chlorocebus_sabaeus
- ciona_intestinalis
- ciona_savignyi
- danio_rerio
- dasypus_novemcinctus
- equus_caballus
- felis_catus
- gallus_gallus
- gorilla_gorilla
- homo_sapiens
- macaca_mulatta
- monodelphis_domestica
- mus_musculus
- ornithorhynchus_anatinus
- oryctolagus_cuniculus
- pan_troglodytes
- papio_anubis
- pongo_abelii
- rattus_norvegicus
- xenopus_tropicalis 

The following species lost symbols:
- aspergillus_fumigatus
- musa_acuminata
- zea_mays

**Pull request merged**